index.html

```diff
@@ -7,37 +7,87 @@
    
   </head>
   <body>
+    <header>
+      <nav>
+        <ol>
+          <li id="AboutLink"><a href="#About">About</a></li> 
+          <li id="ExperienceLink"><a href="#Experience">Experience</a></li>
+          <li id="SkillsLink"><a href="#skills"> Skills</a></li>
+          <li id="ProjectsLink"><a href="#projects"> Projects</a></li>
+          <li id="ConnectLink"><a href="#connect"> connect</a></li>
+        </ol>
+      </nav>
+    </header>
 
     <section id="my name" style="text-align: center;">
-<h1>Farida Hyderi</h1>
-</section>
+      <h1>Farida Hyderi</h1>
+    </section>
 
     <section id="About">
-<h2>About</h2>
-<p id="introduction" >My name is Farida I am very entusiastic to learn new programming languages
-also very much interested in testing </p>
+      <h2>About</h2>
+       <p id="introduction" >My name is Farida I am very entusiastic to learn new programming languages
+        also very much interested in testing </p>
     </section>
 
     <section id = "Experience" >
-<h3>Experience</h3>
-
-      <ul>
-<li id="certification1">Cognizant Agile Methodology from Forage </li>
-<li id="certification2">Accenture Project management Forage </li>
-<li id="certification3">Mastercard - Cybersecurity Job Simulation from Forage</li>
-<li id="Project">Worked on testing projects</li>
-      </ul>
+     <h3>Experience</h3>
+
+      <!-- <ul>
+    <li id="certification1">Cognizant Agile Methodology from Forage </li>
+    <li id="certification2">Accenture Project management Forage </li>
+    <li id="certification3">Mastercard - Cybersecurity Job Simulation from Forage</li>
+    <li id="Project">Worked on testing projects</li>
+      </ul> -->
```

Comment:
In general you don't want to merge files with lines commented out, unless there's a specific reason.